internal/signalfx-agent/pkg/core/config/sources/sources.go

```diff
@@ -36,7 +36,7 @@ type SourceConfig struct {
 	// main disadvantage of watching is slightly greater network and compute
 	// resource usage. This option is not itself watched for changes. If you
 	// change the value of this option, you must restart the agent.
-	Watch *bool `yaml:"watch" default:"true"`
```

Comment:
Q.: the previous changes the values were deprecated and were being ignored, but, this one looks like the default is going to change from `true` to `false`. What am I missing in this picture?

Reply:
Is this deprecated but there is no comment reflecting that?

Reply:
this field is part of a struct which is used in only one place, where it is marked deprecated:
https://github.com/signalfx/splunk-otel-collector/blob/main/internal/signalfx-agent/pkg/core/config/config.go#L170